src/stories/1.index.stories.jsx

```diff
@@ -119,6 +120,16 @@ storiesOf('Widgets/Animations/Spinner', module).add('default', () => (
   <Spinner />
 ))
 
+storiesOf('Widgets/WalletButton', module)
+  .add('default', () => (
+    <WalletButton onClick={() => {}}> Add Wallet Contract </WalletButton>
+  ))
+  .add('type submit', () => (
```

Comment:
nit: not a bad idea, but this use case doesn't change the outcome in storybook at all, so ultimately unnecessary.